app/models/aces/mec_check.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Aces
+  # MecCheck to see if person is in Medicaid System
+  class MecCheck
+    include Mongoid::Document
+    include Mongoid::Timestamps
+
+    # Unique Identifier(hbx_id) of the application.
```

Comment:
We aren't using the hbx id, it is the application id, so will want to adjust this comment